caesarCipher.test.js

```diff
@@ -0,0 +1,8 @@
+import caesarCipher from './caesarCipher';
```

Comment:
- [ ] When I test the string "If he had anything confidential to say" against your caesarCipher function:

I got a failing test:

```js
// ohk > oHk
Expected: "Pm ol ohk hufaopun jvumpkluaphs av zhf"
Received: "Pm ol oHk hufaopun jvumpkluaphs av zhf"
```